test/e2e/network/service_latency.go

```diff
@@ -225,6 +226,9 @@ func newQuerier() *endpointQueries {
 // nice properties like:
 //   - remembering an endpoint if it happens to arrive before it is requested.
 //   - closing all outstanding requests (returning nil) if it is stopped.
+//
+// Note that this test case uses a Service with a single endpoint, which is a single-stack
```

Comment:
where is the single-stack requirement? this test works on dual stack already https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20dual,%20master&include-filter-by-regex=Service%20endpoints%20latency%20should%20not%20be%20very%20high

Reply:
Oh, I guess saying it's a single-stack Pod is inaccurate; the _Service_ is single-stack, becuase `ipFamilyPolicy` defaults to `SingleStack` even in dual-stack clusters.

Anyway, point being: the Service has exactly 1 endpoint IP, so it can't hit any of the cases that would result in it having more than 1 EndpointSlice.